src/utils/misc.cpp

```diff
@@ -436,6 +436,41 @@ QByteArray Utils::Misc::startSynchronousProcess(const QString &executablePath,
     return result;
 }
 
+/**
+ * Starts a synchronous process and waits for its result
+ *
+ * @param cmd the terminal command containing all input and output information
+ * @return true on success, false otherwise
+ */
+bool Utils::Misc::startSynchronousResultProcess(TerminalCmd &cmd) {
+    QProcess process;
+
+    // start executablePath synchronous with parameters
+#ifdef Q_OS_MAC
+    process.start("open", QStringList()
+                              << executablePath << "--args" << parameters);
```

Comment:
this probably will fail to compile on mac

Reply:
This was copied from `startSynchronousProcess`?

Reply:
yes but the variables, `executablePath` and so on are not declared, so it will not compile

Reply:
See: https://travis-ci.org/github/pbek/QOwnNotes/jobs/688374906#L808 :grin:

Reply:
`cmd.executablePath` and `cmd.parameters` will work better